src/platform/plugins/private/vis_types/gauge/public/plugin.ts

```diff
@@ -19,7 +19,7 @@ import { setDataViewsStart } from './services';
 
 /** @internal */
 export interface VisTypeGaugeSetupDependencies {
-  visualizations: VisualizationsSetup;
```

Comment:
are this changes necessary ? wont this plugins also get disabled automatically as they list visualizations as required plugin ?

Reply:
to clarify: i suggest to not change visualizations in theese plugins to optional, but keep it as required, which will automatically disable this plugins (as they only extend the visualizations plugins registry)

Reply:
this goes for all plugins in vis_types folder, shared/vis_types folder, vis_type_markdown, possibly others

Reply:
reverted all of these. I am so confused RN how this happened that initially I needed it. Probably I added enabled to config in visualizations at the end and that solved all the issues... 
anyway - it seems to be working just fine now